tests/default/_core/msearch_template.yaml

```
@@ -0,0 +1,47 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test _termvectors and _msearch/template APIs.
+prologues:  
+  - path: /_bulk
+    method: POST
+    parameters:
+      refresh: true
+    request:
+      content_type: application/x-ndjson
+      payload:
+        - {create: {_index: books, _id: book1}}
+        - {author: Harper Lee, title: To Kill a Mockingbird, year: 1960}
+        - {create: {_index: books, _id: book2}}
+        - {author: Elizabeth Rudnick, title: Beauty and the Beast, year: 1991}
+        - {create: {_index: books, _id: book3}}
+        - {author: George Orwell, title: '1984', year: 1949}
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+
+chapters:
+  - synopsis: Perform a multi-search template query using a GET request.
+    path: /_msearch/template
+    method: GET
```

Since https://github.com/opensearch-project/opensearch-api-specification/pull/724 you can do this,  no need to duplicate the test with identical parameters.

```suggestion
    method:
       - GET
       - POST
```